test-utils/test/shared/act.js

```diff
@@ -0,0 +1,47 @@
+import { options, createElement as h, render } from 'preact';
+import { useState } from 'preact/hooks';
+
+import { setupScratch, teardown } from '../../../test/_util/helpers';
+import { act } from '../../src';
+
+/** @jsx h */
+describe('act', () => {
+
+	/** @type {HTMLDivElement} */
+	let scratch;
+
+	beforeEach(() => {
+		scratch = setupScratch();
+	});
+
+	afterEach(() => {
+		teardown(scratch);
+	});
+
+	it('should reset options after act finishes', () => {
```

Comment:
Idea for another test case to check if previously set options are restored:

```js
it('should restore options.afterPaint', () => {
  const spy = sinon.spy();

  options.afterPaint = spy;
  act(() => null);

  expect(options.afterPaint).to.equal(spy);
  expect(spy).to.not.be.called;
});
```

Reply:
Does this suffice, made two additional tests